### ISSUE TITLE
add `response->content` parameter to `halt`

### ISSUE THREAD

**Issue by JaHIY** [PerlDancer/Dancer2#980]
In dancer1, I can use only `halt` to set content for response.
``` perl
hook before => sub {
    halt('I am halted.');
};
```
But in dancer2, I should use `response->content` to set content for response and `halt`.
``` perl
hook before => sub {
    response->content('I am halted.');
    halt;
};
```
So why not add `response->content` parameter to `halt`? It is so convenient!

**Comment by veryrusty** [comment]
Nice pick up @JaHIY !  There is even an example in Dancer2::Manual: `halt("Unauthorized")`.

@JaHIY do you wish to put together a patch to fix this omission?

**Comment by JaHIY** [comment]
Let me try.

**Comment by veryrusty** [comment]
Resolved by #981. @JaHIY Thanks for again for reporting and providing the Pr.  :dancers:

**Comment by xsawyerx** [comment]
@JaHIY++ !
